code.ts

```diff
@@ -98,20 +143,42 @@ async function startTimer(node: TextNode, seconds: number, template: string, sta
   await figma.loadFontAsync(node.fontName as FontName);
   activeTimer += 1;
 
+  console.log("Timer started / became active");
+
+  var timerID = activeTimer;
+  var keepItRunning = true;
   var secondsToGo = seconds;
-  while(secondsToGo > 0) {
-    var newText = fillUpTimeStringWithTempalte(secondsToInterval(secondsToGo), template);
-    if (startsWithTimer) {
-        newText = "Timer: " + newText;
+  var newText = "";
+
+  while (keepItRunning) {
+
+    // checking if reset was clicked by user and if so resetting all timers
+    if (reset) {
+      secondsToGo = seconds;
+      newText = fillUpTimeStringWithTemplate(secondsToInterval(secondsToGo), template);
+      newText = "Timer: " + newText;
```

Comment:
Not all Timer start with "Timer: "

![Bug](https://user-images.githubusercontent.com/7677738/80486550-a0248a80-895b-11ea-8f9d-3870a219878a.gif)